### DISCUSSION TITLE
Promote Versioned gh-pages docs using a git tag?

### DISCUSSION THREAD

**Discussion by KevinAst** [community/community#21934] [Other Feature Feedback, Questions, & Ideas]
<p>I am using gh-pages for my project docs (ex: <a href="https://action-u.js.org/" rel="nofollow noopener">https://action-u.js.org/</a>). I very much appreciate the gh-pages platform in easily promoting seamless documentation!</p>
<p>With that said, I would very much like to promote versioned documentation (so users have easy access to the docs matching their installed version).</p>
<p>I actually maintain a git tag for my gh-pages branch, so all the resources are available for each doc revision (ex: <a href="https://github.com/KevinAst/action-u/tree/v0.1.0-docs" rel="nofollow noopener">https://github.com/KevinAst/action-u/tree/v0.1.0-docs</a>).</p>
<p>_ <strong>My question is:</strong> _ <em>Is it possible (or could an enhancement be requested) to promote a “versioned” gh-pages site (based on a tagged release of the gh-pages branch)?</em></p>
<p>Certainly I realize that I can maintain versioned directories within my docs, however this is a bit cumbersome, seeing that I have everything needed in this simple tagged version of the docs?</p>
<p><strong>If this is not currently possible</strong> , <em>can you point me to a spot where a request could be made in support of this feature?</em></p>
<p>Your help is greatly appreciated.</p>
<p>&lt;/Kevin&gt;</p>

**Comment by mpboom** [comment, marked as answer]
<p>I don’t know if it’s currently possible. I don’t use GitHub pages myself, although it does make a lot of sense to me that this would be possible.</p>
<p>However, if you want to proceed and turn it into a feature request you should do that here: <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a>.</p>
